process/resultsParser/resultsParser.go

```diff
@@ -1,40 +1,34 @@
 package resultsParser
 
 import (
-	"bytes"
 	"encoding/base64"
 	"encoding/hex"
 	"fmt"
 	"strings"
 
-	"github.com/multiversx/mx-chain-core-go/core"
 	"github.com/multiversx/mx-chain-core-go/data/transaction"
 	logger "github.com/multiversx/mx-chain-logger-go"
 )
 
-var log = logger.GetOrCreate("api/gin")
+var log = logger.GetOrCreate("resultsParser")
+
+const atSeparator = "@"
 
 // ResultOutcome encapsulates data contained within the smart contact results.
 type ResultOutcome struct {
-	ReturnCode    ReturnCode      `json:"returnCode"`
-	ReturnMessage string          `json:"returnMessage"`
-	Values        []*bytes.Buffer `json:"values"`
+	ReturnCode    string   `json:"returnCode"`
+	ReturnMessage string   `json:"returnMessage"`
+	Values        [][]byte `json:"values"`
 }
 
 // ParseResultOutcome will try to translate the smart contract results into a ResultOutcome object.
```

Comment:
change comment because this function parse also log events

Reply:
modified.